docs/rules/minimal.md

```diff
@@ -12,6 +12,8 @@ Errors:
 - [no-server-variables-empty-enum](./no-server-variables-empty-enum.md)
 - [no-unresolved-refs](./no-unresolved-refs.md)
 - [spec](./spec.md)
+- [stepId-unique](./arazzo/stepId-unique.md)
+- [workflowId-unique](./arazzo/workflowId-unique.md)
```

Comment:
Yeah.

Reply:
Yes, we really, really need this. We also really need 3.0 and 3.1 especially as 3.2 is soon and will require some more (minor) alterations.

Reply:
We know what format of description we have, so we could create recommended/etc rulesets for each option and have them be the zero-config defaults.

Reply:
I'm not sure we need separate rulesets for each specification type and flavor. Redocly CLI is smart enough to detect which specification it's working with and applies only the appropriate rules. The only issue arises with Spot-specific rules which contradict (to certain extent) with the Arazzo spec. Therefore, it might be a good idea to introduce only one new spot ruleset. If you're interested in Spot-specific rules, you can use config like the following:

```yaml
extends:
- recommended
- spot
...
```

Reply:
But they're additive. I can't extend recommended for my OpenAPI and minimal for my Arazzo, I can only have everything, or manually build up the rulesets under the per-format sections.

I'm adding documentation with copy/paste samples for how to recreate the rulesets yourselves in config as a workaround - we can update the docs again if we improve the feature in future.

Reply:
I think this is covered by #1724 so I'm resolving this thread and we can pick up the discussion outside this PR